### ISSUE TITLE
All hotkey commands disappeared in 1.5.4

### ISSUE THREAD

**Issue by cmeeren** [mii-key/obsidian-links#17]
Reverted to 1.5.3, they're back then.

**Comment by mii-key** [comment]
hi,
Thanks for using the plugin.
I've checked update from 1.5.3 to 1.5.4, however I couldn't reproduce the issue you described.
Hotkeys are stored in `<your-vault>/.obsidian/hotkeys.json` file that should not be effected neither by update nor by the plugin.
I appreciate if you could be more specific on what hotkeys stopped working and provide me with the steps to reproduce the issue.

**Comment by cmeeren** [comment]
I don't think the hotkeys themselves disappeared from that file, because they started working again when I downgraded.

The issue is that in the command palette as well as in the key map settings, all commands from the Links plugin simply weren't there, just like if I hadn't installed the plugin (but I verified that it was enabled, and even disabled and enabled it, and restarted Obsidian).

For all i know, it could be an issue with the interaction with other plug-ins. I just updated Settings Search, there was a mention in the changelog about issues with other plug-ins (not sure if the issue would manifest in Settings Search or other plug-ins).
